### ISSUE TITLE
[Settings> Color Schemes/Actions]: Accessible name is not descriptive for 'Add New', 'Rename', 'Add new', 'Edit', 'Accept', 'Cancel' and 'Delete' buttons inside Color Schemes & Actions page.

### ISSUE THREAD

**Issue by v-rpundir** [microsoft/terminal#12010]
### Windows Terminal version

1.12.3472.0

### Windows build number

10.0.22504.1010

### Other Software

Test Environment:
OS: Windows 11 Version Dev (OS Build 22504.1010)
App: Windows Terminal Preview
Screen Reader: Narrator
Tool: Accessibility Insights for Windows Version 1.1.1741.1

### Steps to reproduce

**Repro Steps:**
1. Open Windows Terminal.
2. Open Settings page using 'Ctr+,'.
3. Open Narrator using 'Win+Ctrl+ Enter' key. 
4. Now navigate to 'Add New', 'Rename', 'Add new', 'Edit', 'Accept', 'Cancel' and 'Delete' buttons inside Color Schemes & Actions page and observe the issue.

**User Experience:**
Screen Reader users UX will not be good if control name is not descriptive.

**Guideline Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/name-role-value

**Attachments:**
[Accessible name is not descriptive for buttons.zip](https://github.com/microsoft/terminal/files/7751151/Accessible.name.is.not.descriptive.for.buttons.zip)

### Expected Behavior

Accessible name should be descriptive i.e. 'Add New Color Schemes', 'Rename Color Schemes', 'Add new Actions', 'Edit Actions', 'Accept Actions', 'Cancel Actions' and 'Delete Actions' buttons inside Color Schemes & Actions page.

### Actual Behavior

Accessible name is not descriptive for 'Add New', 'Rename', 'Add new', 'Edit', 'Accept', 'Cancel' and 'Delete' buttons inside Color Schemes & Actions page.
i.e. For 'Edit' button Screen Reader is only announcing 'Edit' button.

**Observation:** Same issue repro with NVDA Screen Reader.

**Comment by zadjii-msft** [comment]
Hey @PankajBhojwani, does your entire rewrite of the color schemes page address this? (out of curiosity).

This also applies to the actions page, but I just wanted to check the schemes page.

**Comment by PankajBhojwani** [comment]
> Hey @PankajBhojwani, does your entire rewrite of the color schemes page address this? (out of curiosity).

Not as of now!
